### PR TITLE
Add authentication

### PR DESCRIPTION
**Pull request by Taracuda**
This pull request covers story 1.15 and 1.23

Using Amazon Cognito.

### CODE REVIEW

**Comment by conhur** [comment]
I can see the login page that ask for a username and password or to set up an account.

**Review by firicano33 — COMMENTED** [review]
Hi Tara.  Looks good!

File: src/Theme/Theme/IFormContainer.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export interface IFormContainer {
+  width?: string;
```
Comment (piperlincoln) [inline src/Theme/Theme/IFormContainer.ts:2]: Is this question mark meant to be here? Just double checking.

File: src/domain/SignupConfig.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { ISignUpConfig } from "aws-amplify-react/lib-esm/Auth/SignUp";
+
+const SignUpConfig: ISignUpConfig = {
+  hideAllDefaults: true,
+  signUpFields: [
+    {
+      label: 'Email',
```
Comment (piperlincoln) [inline src/domain/SignupConfig.ts:7]: Will they also choose a username or will their email be the username?

**Review by piperlincoln — CHANGES_REQUESTED** [review]
Looks amazing Tara! I just had a few questions. :)

File: src/domain/SignupConfig.ts (hunk 2 above)
Comment (Taracuda) [reply src/domain/SignupConfig.ts:7]: Their email will be their username

File: src/Theme/Theme/IFormContainer.ts (hunk 1 above)
Comment (Taracuda) [reply src/Theme/Theme/IFormContainer.ts:2]: Yes, it has to be there to satisfy Typescript. Sometimes we don't need to width and sometimes we do.

**Comment by piperlincoln** [comment]
Thank you Tara!

**Review by piperlincoln — APPROVED** [review]
Thank you Tara!

**Pull request merged**